torch/utils/cpp_extension.py

```diff
@@ -571,13 +571,31 @@ def _append_sycl_std_if_no_std_present(cflags):
         cflags.append('-sycl-std=2020')
 
 
-def _wrap_sycl_host_flags(cflags):
+def _wrap_sycl_host_flags(host_cflags):
+    cflags = []
     host_cxx = get_cxx_compiler()
-    host_cflags = [
-        f'-fsycl-host-compiler={host_cxx}',
-        shlex.quote(f'-fsycl-host-compiler-options={cflags}'),
-    ]
-    return host_cflags
+    if IS_WINDOWS:
+        for flag in host_cflags:
+            if flag.startswith("-I"):
+                flag = flag.replace("\\", "\\\\").replace("-I", "/I")
+            else:
+                flag = flag.replace("-D", "/D")
+            flag = flag.replace('"', '\\"')
+            cflags.append(flag)
+        cflags = ' '.join(cflags)
+
+        external_include = _join_sycl_home("include").replace("\\", "\\\\")
+        wrapped_host_cflags = [
+            f"-fsycl-host-compiler={host_cxx}",
```

Comment:
The `/std:c++17` which you add on the next line is actually handled by `append_std17_if_no_std_present(host_cflags)` which you also call. Can you drop explicit `/std:c++17` from the line below?